Compressed ccache install information

simonGoldstein opened the pull request:
### Description

The ccache section of install from source was excessively long. It contained some information that wasn't necessary and the section took up a full third of the tutorial. I compressed the section while keeping all of the relevant and important information to increase readability and accessibility.

### Checklist
- [ ] Tested modified webpage locally using the ``build_locally.sh`` script
- [ ] While waiting for someone to review your request, please consider reviewing [another open pull request](https://github.com/ros-planning/moveit/pulls) to support the maintainers

[//]: # "You can expect a response from a maintainer within 7 days. If you haven't heard anything by then, feel free to ping the thread. Thank you!"

rhaschke commented on install/source/index.markdown:
```diff
@@ -100,61 +100,46 @@ Note, every command to ``catkin config --blacklist`` will override the previous
-The following command can be used to do this for `bash`:
+To use `ccache` it must be explicitly configured for use as just installing the `ccache` package is not enough.
+To automatically use `ccache` with all GCC compilations add the `/usr/lib/ccache` directory to your `PATH` before the regular GCC compiler (For systems other than Debian or Ubuntu this may vary).
+It is easy to enable `ccach` on startup in `bash`. For other shells or systems substitute the appropriate commands.
```
```suggestion
It is easy to enable `ccache` on startup in `bash`. For other shells or systems substitute the appropriate commands.
```

rhaschke commented on install/source/index.markdown:
```diff
@@ -100,61 +100,46 @@ Note, every command to ``catkin config --blacklist`` will override the previous
+If setup, `ccache` is used automatically.
 
-New sessions will automatically update the `PATH`, so this is only necessary for your current session.
+To view the statistics on hit ratios and other information use the command bellow.
```
```suggestion
To view the statistics on hit ratios and other information use the command below.
```

simonGoldstein commented:
I fixed the minor spelling mistakes and I checked around for the convention on "note:"s. The old page had bold instead of italic so I changed those to bold once more.

davetcoleman commented on install/source/index.markdown:
```diff
@@ -100,61 +100,45 @@ Note, every command to ``catkin config --blacklist`` will override the previous
-
-On other OS, consult the package manager or software store and search for `ccache`.
+For other OS, consult the package manager or software store and search for `ccache`.
 Refer to the [ccache website](https://ccache.dev) for more information on downloading and installation.
```
the link cache.dev is listed 3x here. I think this sentence referencing it can just be removed.
```suggestion
```

simonGoldstein replied:
Removed

simonGoldstein commented:
@rhaschke I didn't realize that would break anything. I saw that you said it needed some streamlining so I edited the writing to improve the flow. I am now guessing that is not actually what you meant when you commented "Approving after some more streamlining." Thanks for the help regardless, sorry if I messed that up.
